### PR TITLE
Missing Forwardslash

### PR DESCRIPTION
**Pull request by gargoyle**
Fixed missing forward slash between version number and the rest of the URL.

### CODE REVIEW

**Comment by gfosco** [comment]
Paths are provided with the root `/`.  You make a request to `/photos` not just `photos`.  This is at least why it was not in there.  I'm open to reviewing this if you add a function which handles this and allows leaving off the root `/` in request paths.

**Comment by gargoyle** [comment]
Ahh. My bad!

Added a check for a leading `/` and normalize the path accordingly.

File: src/Facebook/FacebookRequest.php [hunk 1]
```diff
@@ -225,7 +225,8 @@ protected function getRequestURL()
     } else {
       $baseUrl = static::BASE_GRAPH_URL;
     }
-    return $baseUrl . '/' . $this->version . $this->path;
+    $normalizedPath = ($this->path[0] == '/') ? $this->path : '/' . $this->path;
```
Comment (yguedidi) [inline src/Facebook/FacebookRequest.php]: `return $baseUrl . '/' . $this->version . '/' . ltrim($this->path, '/');`

**Comment by gargoyle** [comment]
Thanks yguedidi - Much neater! :-)

**Comment by yguedidi** [comment]
You're welcome :)

**Pull request merged**